server/mlabns/templates/map_view.html

```diff
@@ -275,28 +275,29 @@
     </div>
     <div id="map_canvas"></div>
     <div id="footer">
-      <a href="{{ privacy_doc_url }}" class="footer">Privacy policy</a>
```

Comment:
Please don't replace this with `<a href="https://www.measurementlab.net/privacy/" class="footer">Privacy policy</a>`

Reply:
Missed that. Updated the template with the variable from constants in latest commit.

Reply:
This still looks like the measurementlab.net privacy policy?

Reply:
This was updated in a prior commit. The URLs referenced in this template are defined here: 
https://github.com/m-lab/mlab-ns/blob/master/server/mlabns/util/constants.py#L43-L47

Reply:
@critzo and i just talked and figured this out. this is fixed, but the links were re-ordered which is hard to tell in the diff.